tests/kafkatest/services/kafka/kafka.py

```diff
@@ -447,10 +449,29 @@ def create_topic(self, topic_cfg, node=None):
         self.logger.info("Running topic creation command...\n%s" % cmd)
         node.account.ssh(cmd)
 
-        time.sleep(1)
-        self.logger.info("Checking to see if topic was properly created...\n%s" % cmd)
-        for line in self.describe_topic(topic_cfg["topic"]).split("\n"):
-            self.logger.info(line)
+        if describe:
+            time.sleep(1)
+            self.logger.info("Checking to see if topic was properly created...\n%s" % cmd)
+            for line in self.describe_topic(topic_cfg["topic"]).split("\n"):
+                self.logger.info(line)
+
+    def delete_topic(self, topic, node=None):
+        """
+        Delete a topic with the topics command
+        :param topic:
+        :param node:
+        :return:
+        """
+        if node is None:
+            node = self.nodes[0]
+        self.logger.info("Deleting topic %s" % topic)
+        kafka_topic_script = self.path.script("kafka-topics.sh", node)
+
+        cmd = kafka_topic_script + " "
+        cmd += "--zookeeper %(zk_connect)s --delete --topic %(topic)s " % {
```

Comment:
Perhaps we may as well use `--bootstrap-server` since `delete_topic` is a new addition?